packages/make-styles/README.md

```diff
@@ -0,0 +1,16 @@
+# @fluentui/make-styles
```

Comment:
Maybe write some of what you have in the PR description here in the README?

Reply:
Good idea, copy-pasted it 👍